### DISCUSSION TITLE
Kubernetes Issue

### DISCUSSION THREAD

**Discussion by emreakay** [livewire/livewire#3582] [1. Help]
Hi,

I am using kubernetes as production enviroment and using redis as cache.

when i load balance and increse the pod, i cannot get the updated component.
Livewire always sends, first version of the component.

Where does livewire stores the current state?
On the file or redis?

**Comment by joshhanley** [comment]
@emreakay can you explain in more detail what you mean by "cannot get the updated component. Livewire always sends first version of the component"?

Also I'm not familiar with Kubernetes, so when you say increase the pod, do you mean you've added another application server behind your load balancer essentially?

Livewire doesn't store any state on the server, all state is stored in the browser and sent with the request.
Have a read of this for more details https://calebporzio.com/livewire-isnt-actually-live
